multipass/how-to/how-to-set-up-a-graphical-interface.md

```diff
@@ -15,30 +15,43 @@ The graphical desktop can be viewed in various ways. In this document, we descri
 
 The images used by Multipass do not come with a graphical desktop installed. For this reason, a desktop environment must be installed (here we use `ubuntu-desktop` but there are as many other options as flavors of Ubuntu exist), along with the RDP server (we will use `xrdp` but there are also other options such as `freerdp`). 
 
-To do this, first you need to log in to the running Multipass instance:
+To do this, first you need to choose a Multipass instance. List your instances:
 
 ```plain
-$ multipass shell headbanging-squid
+multipass list
+```
+
+Sample output:
+
+```plain
+Name                    State             IPv4             Image
+headbanging-squid       Running           10.49.93.209     Ubuntu 22.04 LTS
+```
+
+Next log in to the running Multipass instance:
+
+```plain
+multipass shell headbanging-squid
 ```
 
 Once inside the instance, run the following commands to install `ubuntu-desktop` and `xrdp`:
 
 ```plain
-$ sudo apt update
-$ sudo apt install ubuntu-desktop xrdp
+sudo apt update
```

Comment:
Thanks for spotting these extra `$` signs! They haven't been cleaned up consistently from all pages yet, this is a good contribution while you're editing this page :)

Reply:
my pleasure, once I have the right Trust level maybe I can script it. It would be easier to do locally in a GitHub fork, but I assume that it is a lot of work to sync back and forth between discourse and Git.

Reply:
> my pleasure, once I have the right Trust level maybe I can script it. It would be easier to do locally in a GitHub fork, but I assume that it is a lot of work to sync back and forth between discourse and Git.

Yes, indeed... for the time being, it's mostly a manual process as we don't see the need - for now - to come up with a semi-automated solution. Still, if you spot style guide inconsistencies on a number of pages, let me know and we can process them in batches, if you like :)